src/main/java/ru/iopump/qa/allure/service/JpaReportService.java

```diff
@@ -106,6 +107,20 @@ public void internalDeleteByUUID(UUID uuid) throws IOException {
         return res;
     }
 
+    public Collection<ReportEntity> deleteAllOlderThanDate(LocalDateTime date) throws IOException {
```

Comment:
Create new method in `JpaReportRepository` and delegate  `deleteAllOlderThanDate`  from JpaReportService to `JpaReportRepository`

Reply:
done